src/Sloth.Api/Startup.cs

```diff
@@ -35,8 +58,42 @@ public void ConfigureServices(IServiceCollection services)
                 var xmlPath = Path.Combine(AppContext.BaseDirectory, xmlFile);
                 options.IncludeXmlComments(xmlPath);
             });
+            services.Configure<SlothAuthenticationOptions>(Configuration.GetSection("AuthenticationConfig"));
+            services.AddAuthentication(options => options.DefaultScheme = JwtBearerDefaults.AuthenticationScheme)
+                .AddScheme<AuthenticationSchemeOptions, BaseAuthHandler>(JwtBearerDefaults.AuthenticationScheme,
+                options => { });
+            
+            /*
+            services
+                .AddAuthentication(options =>
+                {
+                    options.DefaultAuthenticateScheme = JwtBearerDefaults.AuthenticationScheme;
+                    options.DefaultChallengeScheme = JwtBearerDefaults.AuthenticationScheme;
+                    
+
+                })
+                
+                .AddJwtBearer(cfg =>
+                {
+                    cfg.RequireHttpsMetadata = false;
+                    cfg.SaveToken = true;
+                    cfg.TokenValidationParameters = new TokenValidationParameters
+                    {
+                        ValidateIssuer = false,
+                        ValidateAudience = false,
+                        //ValidIssuer = Configuration["JwtIssuer"],
+                        //ValidAudience = Configuration["JwtIssuer"],
+                        IssuerSigningKey = new SymmetricSecurityKey(Encoding.UTF8.GetBytes(Guid.Empty.ToString())),
+
+                        ClockSkew = TimeSpan.Zero // remove delay of token when expire
+                    };
+                });*/
 
             AddChatServices(services);
+            services.AddTransient<IPasswordHasher<User>, PasswordHasher<User>>();
```

Comment:
create Bootstrap class in `Auth` project with an extension method instead of configuring it right here

Reply:
I am going to do this in next step and move token actions to separate class